examples/compat/ggplot/density.py

```diff
@@ -9,4 +9,4 @@
 
 plt.title("Density ggplot-based plot in Bokeh.")
 
-mpl.to_bokeh(name="density")
+mpl.show(mpl.to_bokeh(name="density"))
```

Comment:
I would use directly `show` from `plotting` (importing it at the top of each example), I mean...

`show(mpl.to_bokeh(name="density"))`

and the same for all the examples... `show` would be something bigger than plotting in the future, so it makes sense to use it here too...

Btw, I did not put any `show` function inside mpl,  `mpl.show` is working because of the show import at the top (the one I comment above).

Finally using `mpl.show(mpl...` is too much `mpl`-verbose :wink: